recipes/itak/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+cp iTAK.py $PREFIX/bin/
+cp itakm.py ${SP_DIR}
+mkdir $PREFIX/share/itak
```

Comment:
Is this still needed?

Reply:
> Is this still needed?

yes, I didn't write copy script in yaml file for it does not work on my local. So, I use a bash to instead it

Reply:
This bash script is used for copying itak to bin folder and corresponding module itakm.py to python library folder. Then, it also create itak folder in share folder for database files.  Please let me know if it is ok to use a bash script to copy files. Thankyou